CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## v1.7.0 (06/15/2020)
+
+**Common**
+
+* Corrected various formatting to be compliant with the coding style.
+* Fixed the issue with `olp::client::TaskContext`. Now, it does not trigger cancellation after a successful operation.
+* Added the `default_cache_expiration` added to `olp::client::OlpClientSettings`. Now, you can can control the time how long the downloaded data is considered valid. Once the data has expired it will not be returned to the user upon requests and gradually removed from the cache when new data is added.
+* Added the `OLP_SDK_DISABLE_DEBUG_LOGGING` CMake option. When enabled, the SDK does not print any `TRACE` or `DEBUG` level log messages.
+* Added the `compression` setting to `olp::cache::CacheSettings`. You can now pass the flag to the underlying database engine to enable or disable the data compression.
+* Added the `Compact` API to `olp::cache::DefaultCache`. It is used to optimize the underlying mutable cache storage.
```

Comment:
Please also mention that there is a thread starting to compact in the background once the fs size of the cache exceeds the specified maximum allowed size.